src/main/java/ArraySortedBag.java

```diff
@@ -79,46 +92,50 @@ private int indexOf(T target) {
         }
         return NOT_FOUND;
     }
-//    private int indexOf(T target) {
+
+    /**
+     * Linear search through the bag to find the index
+     * of where the element should be inserted into the
+     * sorted bag.
+     *
+     * @param element Item to be inserted
+     * @return Index of where the element should be inserted
+     */
+    private int findInsertIndex(T element) {
+        int searchIndex = 0;
+        for (T bagElement : this) {
+            if (element.compareTo(bagElement) <= 0) {
+                break;
+            }
+            searchIndex++;
+        }
+        return searchIndex;
```

Comment:
Minor, can do `return searchIndex;` instead of the `break`.

Reply:
:facepalm: